components/Head.js

```diff
@@ -7,6 +7,7 @@ export default class extends React.Component {
       <Head>
         <meta name='viewport' content='width=device-width, initial-scale=1' />
         <link rel='manifest' href='static/manifest.json' />
+        <link href='https://fonts.googleapis.com/css?family=Open+Sans|Roboto' rel='stylesheet' />
         <title>Resistance Calendar</title>
```

Comment:
Yup, I went with a CDN since the font selection hasn't been finalized yet.  The designers will be able to swap CDNs as necessary until they're 100% sure of which ones we should stick with.